### ISSUE TITLE
Create template for Reviewing the alt text on all pages

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/website#2515]
### Overview
We need to create a template so that we can easily make issue for the design team to check each page on the site in a consistent way.

### Action Items
- [x] Create template draft in comment below
- [x] Talk to Matthew to find out if dev is going to make this list
  - [x] revise after Matthew conversation
- [x] review with Bonnie
- [x] have Bonnie help you make it into a template on the site
- [x] Make for all relevant issues
- [ ] List the issues under resources with checkboxes

### Resources/Instructions
- [Entire worksheet Google Sheet](https://docs.google.com/spreadsheets/d/1abMlPOX88zHTV29E4BFxAYwahCxgJN28Mn95cXr8jak/edit#gid=308160713)
- Issue we are modeling this template after: https://github.com/hackforla/website/issues/2354

**Comment by ExperimentsInHonesty** [comment]
# Template draft
---
title: Alt Text Audit - Design
description: Template issue to create page audit issues for designers
default title: Review the alt text for [insert page name here] page images and ensure that they adhere to WCAG
default labels: design, feature: Accessibility, size: small
---

### Overview
We need to ensure that alt text for images adhere to Web Content Accessibility Guidelines (WCAG). For this issue, we will tackle the images under the  [NAME OF PAGE] page.

#### Details
Developers identified that some alt texts in the [NAME OF PAGE] page should be reviewed because they are either inaccurate or too long. Ideally, alt text should be descriptive but short and succinct. According to the WCAG page, an accompanying alt text should be short. Additionally, the Functional Accessibility Evaluator (FAE) recommends for alt text to be no more than 100 characters.

### Action Items 
- [ ] Make a copy of the tab Template [Page Name]- [Status] 
- [ ] With the copied tab replace [Page Name] with the page you are reviewing and [Status] with "Audit in Progress". Don't forget to remove brackets "[ ]"
- [ ] Add a link of the page your working on to the Resources/Instructions section of this issue at the bottom where it says "Google sheet Tab for your page".
- [ ] For each item in the list, identify if the alt text needs to be changed or not using the recourses below and fill out the google sheet.
- [ ] When completed change your google sheet tabs status to Audit Complete
- [ ] Review with Design Lead or another designer/researcher
- [ ] Mark issue as done

Development
- [ ]  Create issues for that particular alt text should be created (preferably labelled good first issue) for new developers to fix

### Resources/Instructions
[NAME of PAGE on the hackforla.org site](url)
[WCAG section on alt text](https://www.w3.org/WAI/WCAG21/Techniques/html/H37.html)
[FAE rule 4: alt text should be less than 100 characters](https://fae.disability.illinois.edu/rulesets/IMAGE_4_EN/)
[Google sheet instructions](https://docs.google.com/spreadsheets/d/1abMlPOX88zHTV29E4BFxAYwahCxgJN28Mn95cXr8jak/edit#gid=308160713)
[Google sheet template](https://docs.google.com/spreadsheets/d/1abMlPOX88zHTV29E4BFxAYwahCxgJN28Mn95cXr8jak/edit#gid=129703697)
Google sheet Tab for your page

**Comment by IsaacDesigns** [comment]
Progress: In progress
Blockers: Need to know if Matt wants to continue with the current way or create templates
Availability: 6 hours
ETA: December 5th

**Comment by ExperimentsInHonesty** [comment]
@macho-catt I think there might be some confusion on this issue https://github.com/hackforla/website/issues/2515.  Isaac Cruz and I have already made the [template for designers.](https://github.com/hackforla/website/issues/2515#issuecomment-978213787)  What we  need to know from you, is: Is development going to make a list of all the images on a page and put them in the spreadsheet (each page on its own tab) and then design annotates it?  Or do you want design to make the list of images in the spreadsheet as well as annotate it?

**Comment by macho-catt** [comment]
@ExperimentsInHonesty I spoke to @IsaacDesigns and we decided we will split up the pages on the website and update the spreadsheet together.

**Comment by IsaacDesigns** [comment]
Progress: I have confirmed with @macho-catt that Design and Development will split up the pages and design will annotate all of them.
Blockers: Waiting for @macho-catt to make a list of pages for Design to add to the spreadsheet.
Availability: 4 hours
ETA: NA

**Comment by macho-catt** [comment]
List of pages (12 in total):

Devs
- [About page](https://www.hackforla.org/about/)
- [Projects page](https://www.hackforla.org/projects/)
- [Events page](https://www.hackforla.org/events/)
- [Getting Started](https://www.hackforla.org/getting-started)
- [Join Us page](https://www.hackforla.org/join)
- [Sitemap](https://www.hackforla.org/sitemap/)

Design
- [Toolkit](https://www.hackforla.org/toolkit/)
- [Donate](https://www.hackforla.org/donate/)
- [Wins](https://www.hackforla.org/wins/)
- [Citizen Engagement](https://www.hackforla.org/citizen-engagement)
- [Communities of Practice](https://www.hackforla.org/communities-of-practice)
- [Credits](https://www.hackforla.org/credits/)

**Comment by macho-catt** [comment]
https://docs.google.com/spreadsheets/d/1abMlPOX88zHTV29E4BFxAYwahCxgJN28Mn95cXr8jak/edit#gid=2047679475

About page @macho-catt 
Projects page @JimGeist, @anthonysim 
Events page @lopezpedres, @SAUMILDHANKAR (completed)
Getting Started @R-Tomas-Gonzalez 
Join Us page @JessicaLucindaCheng 
Sitemap @Johnnie007

**Comment by anthonysim** [comment]
Projects Page - bottom half only

https://docs.google.com/spreadsheets/d/1abMlPOX88zHTV29E4BFxAYwahCxgJN28Mn95cXr8jak/edit#gid=68583811

Availability for this week:
Wed (12/8): 5 - 7 PM
Fri (12/10): 5 - 7 PM
Sat (12/11): 11 - 1 PM

My estimated ETA for completing this issue: 12/8/2021

**Comment by JessicaLucindaCheng** [comment]
Completed listing the current alt text for images on the Join Us webpage in the following spreadsheet:
https://docs.google.com/spreadsheets/d/1abMlPOX88zHTV29E4BFxAYwahCxgJN28Mn95cXr8jak/edit#gid=390597843

**Comment by IsaacDesigns** [comment]
Will go over on how to do these tasks with the design team at 12/9 meeting. Then create issues for each page after @ExperimentsInHonesty creates template.

**Comment by IsaacDesigns** [comment]
Progress: 80% complete
Blockers: Need @ExperimentsInHonesty to turn this into a template
Availability: 4 hours
ETA: NA

**Comment by IsaacDesigns** [comment]
I created Issues to audit the alt text for each of these pages:

Design
- [Toolkit](https://www.hackforla.org/toolkit/)
- [Donate](https://www.hackforla.org/donate/)
- [Wins](https://www.hackforla.org/wins/)
- [Citizen Engagement](https://www.hackforla.org/citizen-engagement)
- [Communities of Practice](https://www.hackforla.org/communities-of-practice)
- [Credits](https://www.hackforla.org/credits/)

Will need to create issues to audit the alt text for the rest of these pages that were given to devs (After devs input the images in their [Google sheet tab](https://docs.google.com/spreadsheets/d/1abMlPOX88zHTV29E4BFxAYwahCxgJN28Mn95cXr8jak/edit#gid=308160713).

Devs
- [About page](https://www.hackforla.org/about/)
- [Projects page](https://www.hackforla.org/projects/)
- [Events page](https://www.hackforla.org/events/)
- [Getting Started](https://www.hackforla.org/getting-started)
- [Join Us page](https://www.hackforla.org/join)
- [Sitemap](https://www.hackforla.org/sitemap/)

**Comment by ExperimentsInHonesty** [comment]
@IsaacDesigns Did this template get made

**Comment by IsaacDesigns** [comment]
@ExperimentsInHonesty I need you to show me how to make the template when you get the chance.

**Comment by ExperimentsInHonesty** [comment]
Sorry for the delay.  Have added you as an admin.
